README.md

```diff
@@ -63,10 +64,10 @@ You will also need to ensure you install some devDependencies for testing purpos
     * Jest-sorted
     * Supertest
```

Comment:
Again NPM install will take care of this as the dependencies and dev dependencies are listed in your package.json